src/libsystemd/sd-bus/sd-bus.c

```diff
@@ -4198,7 +4198,6 @@ _public_ int sd_bus_get_description(sd_bus *bus, const char **description) {
         assert_return(bus, -EINVAL);
         assert_return(bus = bus_resolve(bus), -ENOPKG);
         assert_return(description, -EINVAL);
-        assert_return(!bus_origin_changed(bus), -ECHILD);
```

Comment:
mmh, but is the object going to be valid? are we 100% sure?

Reply:
A fork does not invalidate memory. (If it did, then even this check would be impossible.)  So the object is valid, we just don't want to use it.

Reply:
ok